pytests/tests/test_install.py

```diff
@@ -22,7 +26,46 @@ def setup_test(utils):
 def teardown_test(utils):
     pkgname = utils.config["mulversion_pkgname"]
     utils.run(['tdnf', 'erase', '-y', pkgname])
-
+    if os.path.isdir(WORKDIR):
+        shutil.rmtree(WORKDIR)
+
+def create_repo(utils):
```

Comment:
Please move all the checksum tests into a new `test_checksum.py` (or similar) file.